### ISSUE TITLE
Weird Private Key implementation

### ISSUE THREAD

**Issue by schrnz** [nimiq-network/core#53]
In contrast to the PublicKey class that is well-used across the core, the PrivateKey implementation seems to be a copy-pasted version (actually in line [12](https://github.com/nimiq-network/core/blob/master/src/main/generic/consensus/primitive/PrivateKey.js#L12) there is PublicKey instead of PrivateKey) that is never really used. 

The problem is that, again in contrast to pub keys, private keys cannot be exported in 'raw' format in the subtle crypto lib we are using. Hence, while the 65 byte size of pub keys makes sense, the private key can only be represented as, e.g.,  'pkcs8', which requires different handling.

Issue #2 also indicates that the private key topic is still somewhat open. 

I think the current implementation is more like a stub. Depending on what we need the class for in the long run (serializing it for wallet portability maybe?), the implementation should reflect the way it is exported from the crypto lib. 

TL;DR: I don't see how to use the PrivateKey class as-is (or where it is used), so either it needs additional explanation or another implementation.
